docs/HowToUseTaskMonitor.md

```diff
@@ -0,0 +1,78 @@
+# How to use Task Monitor
+
+Task Monitor is a kernel thread which monitors the tasks/pthreads.
+It checks if registered tasks/pthreads are alive.
+
+If any task/pthread does not update its active status within registered interval, then Task Monitor will reset the board.
+
+## Contents
+- [Enable Task Monitor](#enable-task-monitor)
+- [Use Task Monitor](#use-task-monitor)
+
+
+## Enable Task Monitor
+To enable Task Monitor, please find four steps as shown below:
+
+### 1. Enable CONFIG TASK MONITOR
+
+In menuconfig, enable **CONFIG_TASK_MONITOR**.
+	```
+	Task Monitor -> Enable Task Monitor
+	```
+
+### 2. Set priority of Task Monitor
+
+In menuconfig, Set **CONFIG_TASK_MONITOR_PRIORITY** value which indicates priority of Task Monitor.
+	```
+	Task Monitor -> Priority of Task Monitor
+	```
+We recommend setting the Task Monitor's priority higher than the registered task/pthread.
+Otherwise, the Task Monitor may not work.
+For example, if a task/pthread has a priority of 100 and a Task Monitor has a priority of 90, when a deadlock occurs, the Task Monitor may not work and the deadlock can persist.
+
+### 3. Set interval for checking alive(sec)
+
+In menuconfig, Set **CONFIG_TASK_MONITOR_INTERVAL** value which indicates interval for checking alive(sec)
+	```
+	Task Monitor -> Interval for checking alive(sec) -> change a number over 0
+	```
+	
+Task Monitor checks the registered tasks/pthreads active status every this interval.
+So if user set CONFIG_TASK_MONITOR_INTERVAL to 5 and registering a task/pthread with interval 7, then the Task Monitor checks the status of the registered task/pthread after 10 seconds instead of 7 seconds.
```

Comment:
So if user sets CONFIG to 5 and registers a ~.

Reply:
modified